src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/Utils/ColorCodeParser.cs

```diff
@@ -6,7 +6,7 @@ namespace Uno.UI.SourceGenerators.XamlGenerator.Utils
 {
 	internal static class ColorCodeParser
 	{
-		public static byte[] ParseColorCode(string colorCode)
+		public static string ParseColorCode(string colorCode)
```

Comment:
- All callers were doing `string.Join(", ", ParseColorCode(...))`, so I just returned the string on the correct form from this method. This avoid the allocation of the small 4-elements byte array.
- This method was allocating a new string via `Substring` unnecessarily.

I don't know if these small improvement will be noticeable, but probably good to have anyway?